build/scraper.js

```diff
@@ -123,23 +148,11 @@ class Scraper {
    */
   async fetchVaultData () {
     const bar = new Progress('Fetching Vault Data', 1)
-    const vaultData = (await get('http://www.oggtechnologies.com/api/ducatsorplat/v2/MainItemData.json')).data
+    const vaultData = (await getJSON('http://www.oggtechnologies.com/api/ducatsorplat/v2/MainItemData.json')).data
 
     bar.tick()
     return vaultData
   }
-
-  applyOverrides (item) {
```

Comment:
any reason we dropped overrides?

Reply:
It's just duplicate code that I forgot to remove in the last refactor. Overrides are being done in parser.js, they shouldn't be in scraper.js